feature-libs/cart/saved-cart/occ/adapters/occ-saved-cart.adapter.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ConverterService, OccEndpointsService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { SavedCartAdapter } from '../../core/connectors/saved-cart.adapter';
+
+@Injectable()
+export class OccSavedCartAdapter implements SavedCartAdapter {
+  constructor(
+    protected http: HttpClient,
+    protected occEndpoints: OccEndpointsService,
+    protected converter: ConverterService
+  ) {}
+
+  create(
+    userId: string,
+    cartId: string,
+    cartDescription: string,
+    cartName: string
+  ): Observable<any> {
+    return this.http.patch(
+      this.getSavedCartEndpoint(userId, cartId, cartDescription, cartName),
+      {}
+    );
+  }
+
+  protected getSavedCartEndpoint(
```

Comment:
change it to getSaveCartEndpoint as getSavedCartEndpoint (name) is for getting a saved cart 

'saveCart' vs 'savedCart' from the default occ saved cart config